backend/src/main/resources/application.yaml

```diff
@@ -37,11 +37,13 @@ graphql:
 okta:
   oauth2:
     issuer: https://hhs-prime.okta.com/oauth2/default
-    client-id: ${OKTA_OAUTH2_CLIENT_ID}
+    # this is overriden by okta-dev, -prod, -test, etc
+    client-id: ${OKTA_OAUTH2_CLIENT_ID:MISSING}
```

Comment:
I don't like having our local flag values for these—I would rather get us back to not having them, and not configuring things accidentally in our non-Okta environments.